cvmfs/server/cvmfs_server_common.sh

```diff
@@ -556,6 +550,19 @@ get_x509_cert_settings() {
   fi
 }
 
+# Parse proxy server for curl command
+get_curl_proxy() {
+  if [ x"$CVMFS_HTTP_PROXY" != x"DIRECT" ]; then
```

Comment:
After switching to `CVMFS_SERVER_PROXY`, the test is now:
```
if [ x"$CVMFS_SERVER_PROXY" != x"" ]; then
```
since `CVMFS_SERVER_PROXY` will be defined only if it contains a proxy URL (i.e. it will never exist with the value `DIRECT` anyway)